vignettes/sirage.R

```diff
@@ -1,50 +1,59 @@
 ## Definition of the time-step and output as "time"
-dt <- user(1)
+dt <- user(0.25)
 initial(time) <- 0
 update(time) <- (step + 1) * dt
 
 ## Core equations for transitions between compartments:
+update(S_tot) <- S_tot - sum(n_SI)
+update(I_tot) <- I_tot + sum(n_SI) - sum(n_IR)
+update(R_tot) <- R_tot + sum(n_IR)
+
 update(S[]) <- S[i] - n_SI[i]
 update(I[]) <- I[i] + n_SI[i] - n_IR[i]
 update(R[]) <- R[i] + n_IR[i]
+update(cumu_inc[]) <- cumu_inc[i] + n_SI[i]
 
 ## Individual probabilities of transition:
-p_SI[] <- 1 - exp(-lambda[i] * dt) # S to I
-p_IR <- 1 - exp(-gamma * dt) # I to R
+p_SI[] <- 1 - exp(-lambda[i]) # S to I
+p_IR <- 1 - exp(-gamma) # I to R
```

Comment:
I think this correction is right – sorry looks like I introduced the error. Can you make the same correction in the SIR odin model?

Reply:
## Individual probabilities of transition:
p_SI <- 1 - exp(-beta * I / N) # S to I
p_IR <- 1 - exp(-gamma) # I to R

In the SIR odin model, the p_SI is slightly different because the N term is used here instead of being used to calculate weighted transmission. Different placement of N but same results essentially.

Reply:
Sorry I was unclear, I was referring to the position of the `dt` here. Discussing on the software paper, it should be in the rate, not the probability

Reply:
i.e. 
```suggestion
p_SI[] <- 1 - exp(-lambda[i] * dt) # S to I
p_IR <- 1 - exp(-gamma * dt) # I to R
```